src/runbenchmark.jl

```diff
@@ -100,9 +100,12 @@ Base.length(ls::LogSpace) = length(ls.r)
 Base.size(ls::LogSpace) = (length(ls.r),)
 Base.axes(ls::LogSpace) = axes(ls.r)
 Base.eltype(::LogSpace) = Int
+
+"""
+"""
 function runbench(
     ::Type{T} = Float64;
-    libs = [:MKL, :OpenBLAS, :BLIS, :Octavian, :Tullio, :Gaius],
+    libs = [:BLIS, :Gaius, :MKL, :Octavian, :OpenBLAS, :Tullio],
```

Comment:
Order of the defaults hopefully doesn't matter, because someone running benchmarks should set up in a way so that clock speeds are constant.

Reply:
Yeah I figured I'd just alphabetize it for readability.